examples/custom-tests/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func main() {
```

Comment:
can this example be a goTest with `TestMain` ?

Reply:
Replaced the existing example with the suggested one.